### PR TITLE
`Development`: Add note about data used in Artemis docs

### PR DESCRIPTION
**Pull request by jakubriegel**
#### General
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://ls1intum.github.io/Artemis/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://ls1intum.github.io/Artemis/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context
It is important to emphasise that Artemis documentation should provide nice experience to its readers and therefore use only real or well prepared data.

### Description
Added a note about personas accounts to be use for preparing screenshots for docs.

### Steps for Testing
Read the docs [here](https://github.com/ls1intum/Artemis/blob/screenshots-in-rtd-docs-data/docs/README.md).

#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2

### CODE REVIEW

**Review by dearjasmina — COMMENTED** [review]
This is a nice explanation; however, please consider removing the passive usage in the text

**Review by dearjasmina — DISMISSED** [review]
thank you for addressing the feedback :)

File: docs/README.md [hunk 1]
```diff
@@ -3,6 +3,25 @@
 We use [Sphinx] for creating the Artemis documentation using [reStructuredText] (RST).
 To get started with RST, check out the [Quickstart] or this [cheatsheet].
 
+## Sample Data & Personas
+Artemis documentation must use realistic examples and personas and must avoid the use of test data. 
+
+Using familiar information in the documentation is crucial because it simplifies the learning process for new users. Real-world scenarios demonstrate to users how to apply specific features within their own context, whereas test data can mislead and fails to reflect real use cases. Realistic examples and personas provide clarity and relevance, ensuring users can effectively understand and utilize Artemis.
+
+Well defined personas are vital for the development process. They not only help readaers to understad the documentation, but also allow developers to better understand Artemis and its users. Many organisations use personas, the two blog posts below contain additional introduction and motivation for the topic:
```
Comment (b-fein) [inline docs/README.md:11]:
```suggestion
Well-defined personas are vital for the development process. They not only help readers to understand the documentation, but also allow developers to better understand Artemis and its users. Many organizations use personas, the two blog posts below contain additional introduction and motivation for the topic:
```

File: docs/README.md [hunk 2]
```diff
@@ -3,6 +3,25 @@
...
+- [Using Personas During Design and Documentation](https://www.uxmatters.com/mt/archives/2010/10/using-personas-during-design-and-documentation.php)
+- [Customer Personas: How to Write Them and Why You Need Them in Agile Software Development](https://community.atlassian.com/t5/App-Central/Customer-Personas-How-to-Write-Them-and-Why-You-Need-Them-in/ba-p/759228)
+
+Screenshots included in Artemis documentation **must** present realistic data. That includes but is not limited to:
+- realistic user, course and exercise names
+- realic text passages, like submissions contents and problem statements
```
Comment (b-fein) [inline docs/README.md:17]:
```suggestion
- realistic text passages, like submissions contents and problem statements  
```

File: docs/README.md [hunk 3]
```diff
@@ -3,6 +3,25 @@
...
+
+Well defined personas are vital for the development process. They not only help readaers to understad the documentation, but also allow developers to better understand Artemis and its users. Many organisations use personas, the two blog posts below contain additional introduction and motivation for the topic:
+- [Using Personas During Design and Documentation](https://www.uxmatters.com/mt/archives/2010/10/using-personas-during-design-and-documentation.php)
+- [Customer Personas: How to Write Them and Why You Need Them in Agile Software Development](https://community.atlassian.com/t5/App-Central/Customer-Personas-How-to-Write-Them-and-Why-You-Need-Them-in/ba-p/759228)
+
+Screenshots included in Artemis documentation **must** present realistic data. That includes but is not limited to:
+- realistic user, course and exercise names
+- realic text passages, like submissions contents and problem statements  
+
+Screenshots included in Artemis documentation **must not** present any test data or server infomation. That includes but is not limited to:
```
Comment (b-fein) [inline docs/README.md:19]:
```suggestion
Screenshots included in Artemis documentation **must not** present any test data or server information. That includes but is not limited to:
```

**Review by basak-akan — APPROVED** [review]
Really nice explanations! Looks good to me 👍

**Review by laadvo — APPROVED** [review]
New documentation looks good!

**Review by dearjasmina — APPROVED** [review]
reapprove

**Pull request merged**